org/mozilla/jss/netscape/security/util/Utils.java

```diff
@@ -374,7 +373,7 @@ public static String base64encode(byte[] bytes, boolean chunked) {
      * @return base-64 encoded data
      */
     public static String base64encodeMultiLine(byte[] bytes) {
-        return new Base64(64).encodeToString(bytes);
+        return Base64.getEncoder().encodeToString(bytes);
```

Comment:
I think we need to use `Base64.getMimeEncoder()` here to generate multi-line output.

Reply:
Ah, good catch, thanks @edewata! I'll update that tomorrow and re-review the rest of these places.